### DISCUSSION TITLE
Error messages

### DISCUSSION THREAD

**Discussion by derrick56007** [h2oai/wave#1058] [Q&A]
Hello All,

Currently working on an Wave App that queries a database and uses the results to populate dropdown choices.
Occasionally the app will hang ( loading animation and no response on page refresh).

Unsure how to debug this.. No errors are outputted in the wave app or on the browser dev-tools console.

Any tips would be appreciated, thanks!

**Comment by mturoci** [comment]
Hi @Derrick56007, please provide a minimal repro code so that we can tell you exactly where the problem is.
